src/lexer.coffee

```diff
@@ -287,6 +303,11 @@ exports.Lexer = class Lexer
     {tokens, index: end} = @matchWithInterpolations regex, quote
     $ = tokens.length - 1
 
+    if prev and not prev.spaced and prev[0] in ['.', '?.', '::', '?::']
+      stringProp = prev[0]
```

Comment:
@zdenko naming maybe `stringProp` -> `precedingAccessor`. Or since it's effectively just being checked for its presence, you could name it `precededByAccessor` and not even bother setting it to `no` in an `else`, eg
```
precededByAccessor = yes if prev and not prev.spaced and prev[0] in ['.', '?.', '::', '?::']
```